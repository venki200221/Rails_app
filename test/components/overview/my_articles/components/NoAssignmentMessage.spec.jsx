import React from 'react';
import { shallow } from 'enzyme';
import '../../../../testHelper';

import NoAssignmentMessage from '../../../../../app/assets/javascripts/components/overview/my_articles/components/NoAssignmentMessage';

describe('NoAssignmentMessage', () => {
  const component = shallow(<NoAssignmentMessage />);
  it('should render', () => {
    expect(component).toMatchSnapshot();
  });
  it('should include a link on how to find an article', () => {
    expect(component.find('a').contains('How to find an article')).toBeTruthy;
  });
  it('should include a link on evaluating articles and sources', () => {
    expect(component.find('a').contains('Evaluating articles and sources')).toBeTruthy;
  });
});
