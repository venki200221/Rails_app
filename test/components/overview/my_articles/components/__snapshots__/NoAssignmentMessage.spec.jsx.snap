// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoAssignmentMessage should render 1`] = `ShallowWrapper {}`;
